### DISCUSSION TITLE
More stats

### DISCUSSION THREAD

**Discussion by QQtin3** [Crownicles/Crownicles#2581] [Nouvelles idées]
Je sais que le backlog est déjà bien complet mais pourquoi pas dans le futur avoir des stats sur le PvE, genre combien de fois tu as combattu un tel monstre, combien de fois tu as perdu, le nombre de tour minimal que ça t'as pris, tes plus grosses victoires / close call, je me dis que ça pourrait être sympa !

**Comment by Voisin1234** [comment]
Je valide plus plus, ça ressemblerait un peu comme ma proposition, mais qui était pour le pvp https://github.com/DraftBot-A-Discord-Adventure/DraftBot/discussions/2264

**Comment by BananePlantain** [comment]
#2136 ^^
quitte a les rassembler en un endroit
